### DISCUSSION TITLE
Overrides to check [xxx]!!!

### DISCUSSION THREAD

**Discussion by peteruoi** [joomla/joomla-cms#47114] [2. Cold Features]
The problem: update joomla from 4.1.5 to 4.2.1
results in 141 ovverides to check!
The majority of them is of the kind 
`<?php
/**
 * @package Helix Ultimate Framework
 * @author JoomShaper https://www.joomshaper.com
 * @copyright Copyright (c) 2010 - 2021 JoomShaper
 * @license http://www.gnu.org/licenses/gpl-2.0.html GNU/GPLv2 or Later
*/

defined ('JPATH_BASE') or die();

require HelixUltimate\Framework\Platform\HTMLOverride::loadTemplate();`

I understand that the majority of these are from a third party template (that is very known) but i have to raise this here cause i don't know if joomla could provide a sollution to this. Something like white listing? 
in the first 30 i checked there were also valid updated overrides like
`
/html/mod_articles_categories/default.php
/html/mod_articles_categories/default_items.php
`
But the whole procedure is so time consuming (just to check not fix) that i am sure we can do better.

### Describe the solution you'd like
At the very least some ux improvements. 
For example the override-check button should be also in the page that we check and not only in the overview of all the ovverides.
Or after opening an override to check when we close it it would be ideal to be in the "height" view that i was. (In my case i have to scroll down 4-5 times justo to find the next). If this is difficult and the ovveride checked button is added inside the opened override we could add also a button check and go to the next override.

Small ux improvements that can win us hours and make this nice feature a lot more user friendly.
Of course if only 4-5 updates were seen instead of 141 that would be the ideal but since 3rd parties are involved i don't know if it can be done

Thanks!

**Comment by brianteeman** [comment]
There are a lot of ux improvements for com_overrides just sat waiting to be merged

**Comment by brianteeman** [comment]
Good idea about the button

**Comment by peteruoi** [comment]
Another great addition would be to be informed if an updated override is updated due to security reasons.

e.g. today i am informed for about 50 ovverides. And i also read that there are some security bugs also. 
1st check goes to ‎/templates/xxx/html/mod_articles_categories/default_items.php
Lots of changes there. 
I don't have any idea if i should adjust or not. If i am informed that there is some security issue there i 'll try to adapt or maybe inform the template developer.

**Comment by peteruoi** [comment]
I would even suggest that red notification in the backend panel about notification should only be red if ovverided detected are due to security. If not a warning color should be used.
